robot.yaml

```
@@ -1,12 +1,24 @@
+# For more details on the format and content:
+# https://github.com/robocorp/rcc/blob/master/docs/recipes.md#what-is-in-robotyaml
+
 tasks:
   Run all tasks:
     shell: python -m robot --report NONE --outputdir output --logtitle "Task log" tasks.robot
```

If we are at it, why not being more elegant:
```suggestion
    robotTaskName: Minimal Task
```
and ensuring we have a _Minimal **T**ask_ task in the _tasks.robot_ file.

Or is this specifically designed to showcase how to customize robot runs?

---

Hmmm.... 
The thing with templates is that users will/should edit it heavily from the get-go.
So locking a specific task name makes that process more cumbersome. 
The shell line works as long as you do not rename the tasks.robot.
So, robot.yaml is cleaner in the robotTaskName -format, but the initial hit to the user might be worse.. this is a bit of 50/50 case.

---

Agree and thank you for the valuable info!